src/ledgered/github.py

```diff
@@ -88,6 +93,47 @@ def current_branch(self, new_branch: str) -> None:
         # invalidating previously fetched info, as they may differ on another branch
         self._manifest = None
         self._makefile = None
+        self._variant_param = None
+        self._variant_values.clear()
+
+    @property
+    def get_variants_cfg(self) -> Tuple[Optional[str], List[str]]:
+        """Returns the Apps VARIANT_PARAM and VARIANT_VALUES"""
```

Comment:
If it's a property you don't need to explicit that the function is a getter in the name, because -as for a external user- it's not a function, it's a property (a field, a variable).

- when you trigger a function: `cfg = app_list.get_variants_configuration()`
- when you access a property: `cfg = app_list.variants_configuration`

```suggestion
    @property
    def variants_configuration(self) -> Tuple[Optional[str], List[str]]:
        """Returns the Apps VARIANT_PARAM and VARIANT_VALUES"""
```

Reply:
TBH I don't really get the meaning of this method, as it is exactly equivalent to `(app_list.variant_param, app_list.variants)`. I'm not sure it's worth adding such a trivial composition.

Reply:
ok, why not. Removed